interceptors/auth/metadata_test.go

```diff
@@ -7,62 +7,62 @@ import (
 	"context"
 	"testing"
 
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/metadata"
```

Comment:
Move this down to the bottom again please

Reply:
Fixed.